src/utils/parseSpecimenBody.js

```diff
@@ -2,15 +2,16 @@ import {safeLoad, CORE_SCHEMA, Type, Schema} from 'js-yaml';
 
 const defaultMapBodyToProps = (parsedBody, rawBody) => parsedBody || rawBody;
 
-const INITIAL_SEPARATOR = '---\n';
-const SEPARATOR = '\n---\n';
+const INITIAL_SEPARATOR = / *--- *\n/;
```

Comment:
Shouldn't this also check for `\t` character?

```regex
/[ \t]*---[ \t]*\n/
```

Reply:
Yes, good spot, thanks!